simde/arm/neon/rsqrts.h

```diff
@@ -37,6 +37,34 @@ HEDLEY_DIAGNOSTIC_PUSH
 SIMDE_DISABLE_UNWANTED_DIAGNOSTICS
 SIMDE_BEGIN_DECLS_
 
+SIMDE_FUNCTION_ATTRIBUTES
+simde_float32_t
+simde_vrsqrtss_f32(simde_float32_t a, simde_float32_t b) {
+  #if defined(SIMDE_ARM_NEON_A64V8_NATIVE)
+    return vrsqrtss_f32(a, b);
+  #else
+    return SIMDE_FLOAT32_C(0.5) * (SIMDE_FLOAT32_C(3.0) - a * b);
```

Comment:
I suggest putting `a * b` in parenthesis.  Whenever you have multiple operations like this, IMHO it's a good idea to explicitly group them.  It makes the code a little bit easier to read since you have spend less mental energy parsing the code… I don't think anyone reading SIMDe is going to have a hard time figuring out which operation comes first, but with parenthesis you really don't have to think about it.

I'll "fix" it when I merge.